src/app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.cpp

```diff
@@ -35,6 +35,7 @@ using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::CameraAvStreamManagement;
 using namespace chip::app::Clusters::CameraAvStreamManagement::Structs;
 using namespace chip::app::Clusters::CameraAvStreamManagement::Attributes;
+using namespace Globals;
```

Comment:
Isn't this actually `chip::app::Clusters::Globals`?

Also, I am not sure using the whole globals namespace is great.  Better would be specific `using chip::app::Clusters::Globals::Whatever` that you need.

Followup is best here; this PR has too many things going on already.

Reply:
Put in the StreamUsageEnum declaration here from the Globals space.